src/extension/src/ActionHandler.py

```diff
@@ -67,29 +68,33 @@ def setup(self, action, log_message):
         self.setup_file_logger(action)
         self.setup_telemetry()
         self.logger.log(log_message)
-        if action == Constants.ENABLE:
-            self.write_basic_status()
+        self.write_basic_status(action)
 
-    def write_basic_status(self):
+    def write_basic_status(self, action):
         """ Writes a basic status file if one for the same sequence number does not exist """
         try:
             # read seq no, if not found, log error and return, as this code opportunistically tries to write status file as early as possible
-            seq_no = self.ext_config_settings_handler.get_seq_no_from_env_var()
-            if seq_no is None:
+            if self.seq_no is None:
                 self.logger.log_error("Since sequence number for current operation was not found, handler could not write an initial/basic status file")
                 return
 
             # check if a status file for this sequence exists, if yes, do nothing
-            if not os.path.exists(os.path.join(self.ext_env_handler.status_folder, str(seq_no) + Constants.STATUS_FILE_EXTENSION)):
-                config_settings = self.ext_config_settings_handler.read_file(seq_no)
+            if os.path.exists(os.path.join(self.ext_env_handler.status_folder, str(self.seq_no) + Constants.STATUS_FILE_EXTENSION)):
+                return
+
+            # if status file for this sequence does not exist and enable is executed, read and use values from config setting file, else do not set those in status file
```

Comment:
Mike and me did an analysis of this change and the decision for now is to not include this change based on these factors: 
- repetitive status file writes for enable (initially the basic status and then again for writing operation)
- Some consideration to not overwrite an existing file in case on a re-enable action. (this is handled in the lines 95-96 here, but handling it later while writing operation will be complex)